sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.1.4 (2020-03-16)
+
+- `@azure/amqp-common` has been updated to increase the default timeout for authorization requests sent to the service from 10s to 60s to reduce the frequency of timeout errors.
+  This release updates `@azure/amqp-common` library to the latest version.
```

Comment:
The term "default timeout" here makes one think that they can customize the timeout value which is not the case

```suggestion
- Updated to use the latest version of `@azure/amqp-common` where the timeout for authorization requests sent to the service is increased from 10s to 60s to reduce the frequency of timeout errors.
```